sample-kotlin/src/main/java/com/fblitho/lithoktsample/lithography/components/StoryCardComponentSpec.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file provided by Facebook is for non-commercial testing and evaluation
+ * purposes only.  Facebook reserves all rights not expressly granted.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package com.fblitho.lithoktsample.lithography.components
+
+import android.graphics.Color
+import com.facebook.litho.*
+import com.facebook.litho.annotations.*
+import com.facebook.litho.widget.Image
+import com.facebook.litho.widget.Text
+import com.facebook.yoga.YogaAlign
+import com.facebook.yoga.YogaAlign.STRETCH
+import com.facebook.yoga.YogaEdge.*
+import com.facebook.yoga.YogaJustify.CENTER
+
+/**
+ * Renders a "story card" with a header and message. This also has a togglable "saved" state.
+ *
+ *
+ * This and the header do most of the interesting stuff for the approximate end state for the lab
+ * activity.
+ */
+@LayoutSpec
+object StoryCardComponentSpec {
+
+  internal val CARD_INSET = 12
+  internal val CARD_INTERNAL_PADDING = 7
```

Comment:
Why not defining those as floats?

Reply:
Also if these are only used inside this ComponentSpec's scope, you can reduce their scope from `internal` to `private`